ci-operator/config/openshift/monitoring-plugin/openshift-monitoring-plugin-main__periodics.yaml

```diff
@@ -0,0 +1,88 @@
+base_images:
+  hypershift-operator:
+    name: hypershift-operator
+    namespace: hypershift
+    tag: latest
+  ocp_4.19_base-rhel9:
+    name: "4.20"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-base-nodejs-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-base-nodejs-openshift-4.19
+  ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.23-openshift-4.19
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  inputs:
+    ocp_4.19_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.19:base-rhel9
+    ocp_builder_rhel-9-base-nodejs-openshift-4.19:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-base-nodejs-openshift-4.19
+    ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19
+  to: monitoring-plugin
+- dockerfile_path: Dockerfile.mcp
+  inputs:
+    ocp_4.19_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.19:base-rhel9
+    ocp_builder_rhel-9-base-nodejs-openshift-4.19:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-base-nodejs-openshift-4.19
+    ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19
+  to: monitoring-console-plugin
+- dockerfile_path: Dockerfile.test
+  to: monitoring-plugin-test
+- context_dir: .
+  dockerfile_path: web/cypress/Dockerfile
+  to: monitoring-ui-tests-runner
+promotion:
```

Comment:
Any reason why we are promoting these images. Are they being used anywhere apart from this CI job. Refer docs https://docs.ci.openshift.org/docs/architecture/ci-operator/#publishing-container-images

Reply:
yeah, makes sense if I am excluding all of them. I've maintained the "same structure" from original config, but let me try to remove this block and rehearse!